service-clients-core/src/main/resources/ServiceJsonSchema.json

```diff
@@ -43,6 +43,9 @@
         },
         "commandName": {
           "type": "string"
+        },
+        "objectMapperClass": {
```

Comment:
Can you remove this and pojojsonschema.json from service-clients-core. This was a miss. They should be present only in service-clients-gen.